changelogs/unreleased/fix-iproto-override-memleak.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/box
+
+- Fixed a memory leak in iproto override (introduced in 3.1).
```

Comment:
This looks critical - the leak happens on execution of every overridden IPROTO request. Please file an issue and reference it here.

Reply:
Created an issue #10477